vm/src/any/memory/spaceSizeMacros.hh

```diff
@@ -9,8 +9,9 @@
 // This must be the size of the largest page of all platforms, to load 
 // snapshots using mmap.
 // sun4c machines use 4096, sun4 use 8192.
+// Cygwin requires 65536.
 // On PPC=, for Altivec, must be multiple of 16. -- dmu 3/04
-# define idealized_page_size 8192
+# define idealized_page_size 65536
```

Comment:
Do we need to change some snapshot versioning number for compatibility reasons because of the change to to `idealized_page_size`?